src/main/java/com/shapesecurity/shift/es2016/semantics/asg/TryCatch.java

```diff
@@ -23,33 +23,28 @@
 
 import java.util.Objects;
 
-public class TryCatchFinally implements Node {
+public class TryCatch implements Node {
 	@Nonnull
 	public final Block tryBody;
 	@Nonnull
-	public final Maybe<Pair<Variable, Block>> catchBody;
+	public final Pair<Variable, Block> catchBody;
```

Comment:
Maybe we want to make the `Variable` optional?

Reply:
Eh, not until 2018 at the earliest, I think.